libc-bottom-half/headers/public/wasi/api.h

```diff
@@ -1300,200 +1300,6 @@ typedef uint32_t __wasi_exitcode_t;
 _Static_assert(sizeof(__wasi_exitcode_t) == 4, "witx calculated size");
 _Static_assert(_Alignof(__wasi_exitcode_t) == 4, "witx calculated align");
 
-/**
```

Comment:
Don't we have tests that verify that this auto-generated file is up-to-date with the witx file?

Reply:
Ah, yes we do, and they're now failing :-}. I'm going to propose we just disable it for now, and just be careful not to make any ABI-breaking changes for now.

Reply:
Sure thing, maybe modify the comment at the top of the file to indicate this.

Reply:
Comment updated.